api/lib/domain/models/AdminMember.js

```diff
@@ -0,0 +1,16 @@
+const PixAdminRole = require('./PixAdminRole');
+
+module.exports = class AdminMember {
```

Comment:
Je lui rajoute `this.disabledAt = disabledAt;`  dont on a besoin dans un autre commit, au moment où on en besoin